okhttp/src/main/java/okhttp3/internal/concurrent/TaskQueue.kt

```diff
@@ -26,13 +26,7 @@ import java.util.concurrent.TimeUnit
  * for its work; in practice a set of queues may share a set of threads to save resources.
  */
 class TaskQueue internal constructor(
-  private val taskRunner: TaskRunner,
-
-  /**
-   * An application-level object like a connection pool or HTTP call that this queue works on behalf
-   * of. This is intended to be useful for testing and debugging only.
-   */
-  val owner: Any
```

Comment:
I could see us adding this back for testing, debugging or monitoring purposes (as previously documented).  Maybe be useful to know which queue is active.  Happy to drop for now, it's easier to add than remove.

Reply:
My hypothesis is that the task info will imply the queue info